example/controller-registration.yaml

```diff
@@ -4,11 +4,14 @@ kind: ControllerRegistration
 metadata:
   name: kupid
 spec:
+  resources:
```

Comment:
@ashwani2k I am not sure if this part is required (it wasn't present before). Can you please if this a change in the gardener extensions behaviour?

The changes look good otherwise.

Reply:
This get introduced I think with make generate. 
Currently we need to change this to work with the controller-deployment and controller-registration semantics. 
This change also requires to update to a newer version of g/g which then needs to be done in a newer PR. 

Can you approve it now.